ykrt/src/jitstate.rs

```diff
@@ -10,15 +10,19 @@ mod jitstate {
 
 #[cfg(feature = "ykd")]
 mod jitstate {
-    use std::{env, sync::LazyLock};
+    use std::{env, fs, sync::LazyLock};
 
-    static JITSTATE_DEBUG: LazyLock<bool> =
-        LazyLock::new(|| env::var("YKD_LOG_JITSTATE").is_ok());
+    static JITSTATE_DEBUG: LazyLock<Option<String>> =
+        LazyLock::new(|| env::var("YKD_LOG_JITSTATE").ok());
```

Comment:
I wonder if the `.ok()`s in this file should be unwraps?

They are very unlikely to fail, but if they do, we'd be confused as to why not logging is showing up.